documentation/examples/column-format/no_matchers.ts

```diff
@@ -0,0 +1,16 @@
+import * as coda from '../../../index';
+
+const pack = coda.newPack();
+
+// BEGIN
+
+// Replace all <text> with your own text
+// a custom column type that you apply to any column in any Coda table.
+pack.addColumnFormat({
+  name: '<format name>',
+  formulaNamespace: '<formulaNamespace of Pack>', // this will be deprecated
+  formulaName: '<the name of a formula to execute using the value in the column>',
+  instructions: '<instruction text that will be visible by Users',
+  // regular expressions used to detect inputs of a certain format type.
```

Comment:
Nit, proper capitalization for all comments visible to users. Maybe,

"Optional, regular expressions that detect urls that are handleable by this column format. When a user pastes a url that matches any of these, the UI will hint to the user that they can apply this format to their column."